templates/dashboards/grafana-dashboard-image-builder-worker-general.configmap.yml

```diff
@@ -2555,11 +2673,52 @@ data:
             "query": "0.95",
             "skipUrlSync": false,
             "type": "constant"
+          },
+          {
+            "current": {
+              "selected": true,
+              "text": [
+                "org-15885990"
+              ],
+              "value": [
+                "org-15885990"
+              ]
+            },
+            "hide": 0,
+            "includeAll": false,
+            "multi": true,
+            "name": "tenant",
+            "options": [
+              {
+                "selected": false,
+                "text": "org-15842261",
+                "value": "org-15842261"
+              },
+              {
+                "selected": false,
+                "text": "org-15877963",
+                "value": "org-15877963"
+              },
+              {
+                "selected": true,
+                "text": "org-15885990",
+                "value": "org-15885990"
+              },
+              {
+                "selected": false,
+                "text": "org-16057323",
+                "value": "org-16057323"
+              }
+            ],
+            "query": "org-15842261, org-15877963, org-15885990, org-16057323",
+            "queryValue": "",
+            "skipUrlSync": false,
+            "type": "custom"
           }
         ]
       },
       "time": {
-        "from": "now-28d",
+        "from": "now-30d",
```

Comment:
Keep the now-28d as default; all our SLOs are calculated across 28d and this way the rest of the graphs (which are time-dependent) show the same time period by default; it's a small thing but still.